cocos/platform/CCFileUtils.cpp

```diff
@@ -1169,22 +1169,17 @@ void FileUtils::getFileSize(const std::string &filepath, std::function<void(long
 
 std::vector<std::string> FileUtils::listFiles(const std::string& dirPath) const
 {
-    std::string fullpath = fullPathForFilename(dirPath);
     std::vector<std::string> files;
+#ifdef UNICODE
```

Comment:
I see tinydir, it checked _MSC_VER define for use WINAPI.
But this function check UNICODE for use WINAPI , before my change.